### PR TITLE
Make replication compatible with pouch

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ swagger-ui
 /.cozy
 *.log
 local.env
+integrations-tests/pouchdb/node_modules
+integrations-tests/pouchdb/replication-target
```
Comment (jinroh) [inline .gitignore]: integration should not be plural here

File: integrations-tests/pouchdb/tests.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+require('longjohn')
+
+var DOCTYPE = 'io.cozy.pouchtestobject'
+var SOURCE = 'http://localhost:8080/data/' + DOCTYPE + '/'
+
+function stackRequest (method, path, body, expect, callback) {
+  var opts = {
+    'method': method,
+    'port': process.env.STACK_PORT || 8080,
```
Comment (jinroh) [inline integrations-tests/pouchdb/tests.js]: may use `COZY_STACK_PORT` to use the same as `cozy-app-dev.sh`

File: scripts/integration.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+go run main.go instances add dev
```
Comment (jinroh) [inline scripts/integration.sh]: maybe use a name other than dev to not conflict. that way we can also do:

```sh
echo "yes" | cozy-stack instances destroy integration-tests
cozy-stack instances add integration-tests
```

Comment (jinroh) [inline scripts/integration.sh]: oh... but that would mean you could not hit "localhost:8080"... meh, nevermind if its too complex

**Comment by aenario** [comment]
@jinroh made the changes

I agree it will be nice to cleanup instances management during tests and use /etc/hosts & :80 -> :8080 iptable to have test env closer to prod. But we should probably make that in another PR !

**Pull request merged**